homeassistant/components/homekit/type_lights.py

```diff
@@ -23,19 +23,19 @@ class Light(HomeAccessory):
     Currently supports: state, brightness, rgb_color.
     """
 
-    def __init__(self, hass, entity_id, name, *args, **kwargs):
+    def __init__(self, hass, entity_id, name, **kwargs):
         """Initialize a new Light accessory object."""
-        super().__init__(name, entity_id, CATEGORY_LIGHT, *args, **kwargs)
+        super().__init__(name, entity_id, CATEGORY_LIGHT, **kwargs)
```

Comment:
Your parent class constructor signature does not have `entity_id`, but you seem to include it here?

```
def __init__(self, name=ACCESSORY_NAME, model=ACCESSORY_MODEL,
           category='OTHER', **kwargs):
```